tools/dreport.d/plugins.d/inventorydata

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+#
+# config: 23 50
+# @brief: Collect inventory persistent data
+#
+
+. $DREPORT_INCLUDE/functions
+
+inventorydir="/var/lib/phosphor-inventory-manager"
```

Comment:
There is already plugin "phosphor-debug-collector/tools/dreport.d/plugins.d/inventory)". How this data is different here. Better update the base plugin instead of creating new one.